### PR TITLE
Issue404

### PR DESCRIPTION
**Pull request by mbab**
+AMDG+

### CODE REVIEW

File: web/cgi-bin/horas/horas.pl [hunk 1]
```diff
@@ -212,11 +212,7 @@ sub resolve_refs {
     elsif ($line =~ /^\s*\!\!(.*)/) {  
       my $l = $1;
       my $suffix = '';
-      if ($l =~ /\{.*?\}/) {
-        $l =~ s/(\{.*?\})//; 
-  	    $suffix = $1;
-		    $suffix = setfont($smallblack, $suffix); 
-      }
+      if ($l =~ s/\{[^:].*?\}//) {$suffix = setfont($smallblack, $&);}
```
Comment (gregordick) [inline web/cgi-bin/horas/horas.pl]: Please avoid `$&` (and ``$` `` and `$'`).  The server is running perl 5.10 and the use of those variables makes matches more expensive on old perls.  The generic way around this is to use an explicit capture group around the whole pattern.

File: web/cgi-bin/horas/specials.pl [hunk 2]
```diff
@@ -16,8 +16,11 @@ sub specials {
   my $lang = shift;		 
   
   $octavam = ''; #check duplicate commemorations
-  my %w = (columnsel($lang)) ? %winner : %winner2; 
-  
+  my %w = (columnsel($lang)) ? %winner : %winner2;
+
+  my $hymntrans = $translate{$lang}{Hymnus};
```
Comment (gregordick) [inline web/cgi-bin/horas/specials.pl]: Maybe this should be

    my $hymntrans = 'Hymnus' || $translate{$lang}{Hymnus};

This is supposing that `$hymntrans` is used again later on (which I imagine that it is, given the last hunk of the patch).

Comment (mbab) [inline web/cgi-bin/horas/specials.pl]: This always gives `'Hymnus'`
If you mean
```
my $hymntrans = $translate{$lang}{Hymnus} || 'Hymnus';
```
it is not necessery as translate always contains default Latin values

Comment (gregordick) [inline web/cgi-bin/horas/specials.pl]: Good points!

**Pull request merged**